modules/nw-egress-router-about.adoc

```diff
@@ -41,6 +62,20 @@ The environment variables determine which addresses the egress-router image uses
 Network Address Translation (NAT) rules are set up so that connections to the cluster IP address of the pod on any TCP or UDP port are redirected to the same port on IP address specified by the `EGRESS_DESTINATION` variable.
 
 If only some of the nodes in your cluster are capable of claiming the specified source IP address and using the specified gateway, you can specify a `nodeName` or `nodeSelector` indicating which nodes are acceptable.
+endif::openshift-sdn[]
+
+ifdef::ovn[]
+An egress router is a pod that has two network interfaces.
+For example, the pod can have `eth0` and `net1` network interfaces.
+The `eth0` interface is on the default cluster network and the pod
```

Comment:
I introduced some early confusion into what _default_ actually means. But there's only a single cluster network, so _default_ is probably inapplicable. Instead, we're calling these other networks, secondary networks. So I'd drop _default_ here.

Instead, you could say that `net1` is on a secondary network.

Reply:
How about calling `net1` something alongside 'the multus' network'? secondary network works as well in any case and it doesn't introduce any references to multus, which may get confusing.

Reply:
I removed "default" from the text. Thanks for the clarification Jason.

For the second item, I'm going to push the foll though it might not be much better: "The `net1` interface is on a secondary network and has an IP address and gateway for that network."